sql/geometry.go

```diff
@@ -235,6 +239,14 @@ func (t GeometryType) Convert(v interface{}) (interface{}, error) {
 			geom, err = WKBToLine(inner[EWKBHeaderSize:], isBig, srid)
 		case WKBPolyID:
 			geom, err = WKBToPoly(inner[EWKBHeaderSize:], isBig, srid)
+		case WKBMultiPointID:
+			return nil, ErrUnsupportedFeature.New("MultiPoint geospatial type")
```

Comment:
These messages should include the value being converted as well

Reply:
ahhh, I see that in other `Convert` methods now. Added a hex representation of the byte[] to the error message.